public/css/widget/state-change.less

```diff
@@ -18,6 +18,11 @@
       margin-left: -.275em;
     }
 
+    &.ball-size-ml {
```

Comment:
The rules here all depend upon the fact that the two balls are of the same size. That's why I suggested to add a new rule that also checks the left ball size.

Reply:
It doesn't matter how diverse the state ball sizes are, the rule looks like this and should be sufficient, I think.
`.state-ball ~ .state-ball`

Reply:
Well, fine, if you think so, combine m + ml for a moment :wink:

Reply:
or heck, even ml + ml